src/core/openthread.cpp

```diff
@@ -1636,6 +1636,21 @@ ThreadError otSetActiveDataset(otInstance *aInstance, const otOperationalDataset
     return error;
 }
 
+bool otIsNodeCommissioned(otInstance *aInstance)
+{
+    otOperationalDataset aDataset;
+
+    otGetActiveDataset(aInstance, &aDataset);
```

Comment:
Ok, I don't think the problem is with the otApi implementation for Windows, but instead the fact that the declaration for that doesn't have the `Thread` namespace, but the definition is in the namespace. I have brought this issue up before. I think we should remove all the `Thread` namespaces from the API definitions.